CONTRIBUTING

```diff
@@ -0,0 +1,87 @@
+Contributing to Anaconda
+=========================
+
+Branching Model
+----------------
+
+The basic premise is that there are the following branches:
+- master
+- unstable
+- <next fedora number>-release
+- <next fedora number>-devel
+
+``Master`` branch never waits for any release-related processes to take place. The spec file will remain there to track dependencies. ``Master`` branch is *not* associated with Fedora Rawhide builds anymore. Its purpose is to function purely as an upstream branch.
+
+The ``unstable`` branch is used for making periodic Anaconda releases for Rawhide (or possibly anyone else wanting to taste the cutting edge).
+
+Concerning current RHEL branches, they are too divergent to integrate into this scheme. Thus, commits are merged onto, and builds are done on the RHEL branches.
+In this case, two pull requests will very likely be needed:
+- one for the ``rhel<number>-branch``
+- one for the ``master`` or ``<fedora number>-devel`` branch (if the change is not RHEL only)
+
+Releases
+---------
+
+For specific Fedora version, releases are made on designated ``-release`` branches and tagged.
+
+Concerning Fedora Rawhide, the release process is slightly different:
+- master is merged onto the unstable branch
+- a release commit is made (which bumps version in spec file) & tagged
+
+Concerning the ``<next Fedora number>`` branches (which could also be called ``next stable release`` if we wanted to decouple our versioning from Fedora in the future):
+- work which goes into the next Fedora goes to ``<next Fedora number>-devel``, which is periodically merged back to ``master``
```

Comment:
Should we mention explicitly that <next Fedora number>-devel is merged to <next Fedora number>-release or is it obvious?